lib/stanford-mods/imprint.rb

```diff
@@ -176,14 +176,19 @@ def decoded_value
               format_date(date.max, date.max.precision)
             ].uniq.compact
 
-            return text.strip if range.empty?
+            return secular_value if range.empty?
 
             range.join(' - ')
           else
-            format_date(date, value.precision) || text.strip
+            format_date(date, value.precision) || secular_value
           end
         end
 
+        # Version of date string using 'BCE' and 'CE' instead of 'BC' and 'AD'.
+        def secular_value
+          text.strip.sub(/B[.]?C[.]?/, 'BCE').sub(/A[.]?D[.]?/, 'CE')
```

Comment:
Do I understand right that this is for changing the text value for any unencoded dates? I'd suggest that's a data remediation problem and not something we should address for display 🤷‍♂️

Reply:
Yep, it is. That would certainly keep things simple, and prevent me needing to port this change to mods_display, where I didn't do it...

Reply:
Reverted these changes to `Imprint`.